build/ci.sh

```diff
@@ -29,6 +29,16 @@ sudo apt-get --yes install clang-"${LLVM_PACKAGE}" \
     libclang-"${LLVM_PACKAGE}"-dev libelf-dev libfl-dev \
     llvm-"${LLVM_PACKAGE}"-dev libz-dev llvm-"${LLVM_PACKAGE}"
 
+sudo apt-get --yes install libc6-dbg
+
+pushd /tmp
+curl -L -O https://sourceware.org/pub/valgrind/valgrind-3.16.1.tar.bz2
+tar xvjf valgrind-3.16.1.tar.bz2
```

Comment:
Maybe you just had this for testing, but I don't think we need verbose output from tar here.